qadence/ml_tools/train_grad.py

```diff
@@ -166,14 +182,20 @@ def loss_fn(model: torch.nn.Module, data: torch.Tensor) -> tuple[torch.Tensor, d
                 best_val_loss, metrics = loss_fn(model, xs_to_device)
 
                 metrics["val_loss"] = best_val_loss
-                write_tensorboard(writer, None, metrics, init_iter)
+                write_tracker(
+                    (writer, None, metrics, init_iter), tracking_tool=config.tracking_tool
```

Comment:
```suggestion
                    writer, None, metrics, init_iter, tracking_tool=config.tracking_tool
```

If you make it variadic as above.

Reply:
Fixed it ineb9a98c